README.md

```diff
@@ -253,9 +286,16 @@ This is used for controlling scenes, and should generally only be used with the
 - Can this scene be deactivated?: Should be left unset if this scene can only be activated and set if this scene can be both activated and deactivated.
 - Deactivate Command: A device command used to deactivate this scene.  Only available if the scene can be deactivated.  Maps to `off` by default.
 
+### SoftwareUpdate
+
+The Software Update trait is used for devices that can have a software update.  It can be controlled by saying things like "Hey Google, Software update {device}." and queried by saying things like "Hey Google, When was {device} last updated?".  It has the following configuration parameters:
+
+- Last Software Update Unix Time Stamp Attribute: The device attribute used to indicate the last Unix time (in seconds) that the update occurred.  Maps to `lastSoftwareUpdateUnixTimestamp` by default.
+- Software Update Command: A device command used to start a software update on the device.  Maps to `softwareUpdate` by default.
```

Comment:
IMO these should have no defaults since there's no matching Hubitat capability.

Reply:
Removed non-matching defaults.